### PR TITLE
Add support for Pride Endpoint in Master

### PR DESCRIPTION
**Pull request by Deivu**

### CODE REVIEW

File: src/lib/Client.js [hunk 1]
```diff
@@ -697,7 +697,17 @@ class IdioticClient {
...
     return this._get("overlays/rainbow", { avatar }).then(body => Buffer.from(body));
   }
-
+  /**
+   * Pride endpoint
+   * @param {string} avatar Image you expect to be used
+   * @param {pride} string Could be one of the FF. agender, aromantic, asexual, bisexual, genderfluid, genderqueer, intersex, lesbian, lgbtq, nonbinary, pansexual, polysexual, straightally, trans
+   * @returns {Promise<Buffer>}
+   */
+  pride(avatar, pride) {
```
Comment (YorkAARGH) [inline src/lib/Client.js]: Any chance you could add the valid options to an array at the top of the file, like I did for the text endpoints, also I removed the `straight` flag as that has some extremely negative connections.

File: typings/index.d.ts [hunk 2]
```diff
@@ -70,6 +70,7 @@ declare module "idiotic-api" {
         public threshold(avatar: string, threshold: number): Promise<Buffer>;
         /* Overlays */
         public rainbow(avatar: string): Promise<Buffer>;
+        public pride(avatar: string, pride: string): Promise<Buffer>;
```
Comment (YorkAARGH) [inline typings/index.d.ts]: Could you replicate what's at the bottom of the file for the valid options for the endpoint please?

File: src/lib/Client.js (hunk 1 above)
Comment (Deivu) [reply src/lib/Client.js]: Will do both once I get home

Comment (YorkAARGH) [reply src/lib/Client.js]: Thanks ^_^

**Comment by Deivu** [comment]
@YorkAARGH Done

**Comment by YorkAARGH** [comment]
Alright, one last thing, and I should have mentioned this, but can you bump the minor version up by 1?

**Comment by Deivu** [comment]
Bumped

**Pull request merged**